core/src/main/scala/org/apache/spark/rpc/netty/NettyRpcEnv.scala

```diff
@@ -184,8 +184,14 @@ private[netty] class NettyRpcEnv(
         // `createClient` will block if it cannot find a known connection, so we should run it in
         // clientConnectionExecutor
         clientConnectionExecutor.execute(new Runnable {
-          override def run(): Unit = {
-            val client = clientFactory.createClient(remoteAddr.host, remoteAddr.port)
+          override def run(): Unit = Utils.tryLogNonFatalError {
+            val client = try {
+                getTransportClient(message.receiver)
```

Comment:
I generally indent things like this when initializing variables. I can change it though.